### PR TITLE
Add lock icon on current attempt letters

### PR DESCRIPTION
**Pull request by joglr**
A lock icon is added on locked letters and on hover, to indicate locked state and ability to lock. 
An unlock icon is also shown on hover, on locked letters, to indicate ability to unlock.

image

### CODE REVIEW

**Comment by joglr** [comment]
#### 📝 Changed routes:


| [**/**](https://ordle-qok01filf-joglr.vercel.app/) |
|:-:|
|<a href="https://ordle-qok01filf-joglr.vercel.app/">Screenshot of /</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/vQgYlzoC4LB77qQqEwxMp6hOkiSvtR9gFs9ZujmqvEGibF3GochWoQV_SRipoXArEpwtXpyVTjeg6Dktab00jg.png">(view full size)</a> |







Commit 19a3d0135f84ce7298c2f3ba75e7e41ba521928d (https://ordle-qok01filf-joglr.vercel.app).

File: src/pages/index.tsx [hunk 1]
```diff
@@ -250,37 +252,74 @@ function Display() {
         </Fragment>
       ))}
       {currentAttempt.map((letter, i) => (
-        <Square
+        <button
```
Comment (rasmusroessum) [inline src/pages/index.tsx:255]: Making this change adds a little change in the height of the div/button. I think this should be fixed

**Review by rasmusroessum — CHANGES_REQUESTED** [review]
Changing from <Square> to <button> gives the UI a little change. Maybe we should put the button in the others or change the Square?

File: src/pages/index.tsx (hunk 1 above)
Comment (rasmusroessum) [reply src/pages/index.tsx:255]: image

image

Comment (joglr) [reply src/pages/index.tsx:255]: We need a proper CSS reset that ensures that this is not a problem.

Comment (joglr) [reply src/pages/index.tsx:255]: image
Seems this is also an issue on the main branch.

**Pull request merged**